app/views/handlers/thank_you.py

```diff
@@ -21,13 +19,11 @@ class ThankYou:
     CENSUS_THANK_YOU_TEMPLATE = "census-thank-you"
     PAGE_TITLE = gettext("Thank you for completing the census")
 
-    def __init__(self, schema):
-        self.session_store: SessionStore = get_session_store()
-        self.session_data: SessionData = self.session_store.session_data
+    def __init__(self, schema: QuestionnaireSchema, session_store: SessionStore):
+        self.session_store: SessionStore = session_store
         self._schema: QuestionnaireSchema = schema
 
-        if not self.session_data.submitted_time:
-            raise NotFound
+        self.session_data: SessionData = self.session_store.session_data
```

Comment:
I don't think a property for `session_data` is necessary. The feedback handler just uses `_session_store.session_data.<property>` when accessing data from the session.